### ISSUE TITLE
Bug : profile photo not visible after changing theme , need to refresh

### ISSUE THREAD

**Issue by NiranjanNlc** [openMF/self-service-app#2486]
**Summary:**

While we change the theme of the app  , the profile photo is not visible in home page 

**Steps to reproduce:**

1. Open the app 
2. change the theme of the app . 
3. observe the profile photo in home page  

**Expected behaviour:**

Profile photo should be clearly visible despite changing profile photo  

**Observed behaviour:**

when we normally open the app, the profile photo is normally visible . But  , when we change the app theme , the profile photo seems go nowhere. The space of the profile photo is observed empty  and the profile photo is invisible  .
**Device and Android version:**

Redm 9A , android version : 10 

**Screenshots:**

https://github.com/openMF/mifos-mobile/assets/25600880/8ffceb86-1a09-4a0f-b9ec-a8530e1e4277

**Comment by NiranjanNlc** [comment]
i will be working on this .

@PratyushSingh07 , please get me this issuses assigned

**Comment by PratyushSingh07** [comment]
But the profile picture re appears when you pull down to refresh

**Comment by NiranjanNlc** [comment]
yeah i noticed  but was not it supposed to appear without refereshng

**Comment by PratyushSingh07** [comment]
> yeah i noticed but was not it supposed to appear without refereshng

not really sure, you can solve it and then maybe we can ask rajan to have a look at it

**Comment by arjesh-vadadoria** [comment]
Hello @NiranjanNlc @PratyushSingh07 , If anyone is not working on it, can I try to solve this?

**Comment by arjesh-vadadoria** [comment]
@PratyushSingh07 if this is not fixed, then can I work on this?

**Comment by NiranjanNlc** [comment]
I am working on this @arjesh-vadadoria .

**Comment by arjesh-vadadoria** [comment]
Okay no problem @NiranjanNlc ,
@PratyushSingh07 if we have any other issues, can you please assign it to me, I can work with intermediate to advanced issues too.

**Comment by NiranjanNlc** [comment]
It was a completely simple fix .
Would you review from your end @PratyushSingh07

**Comment by PratyushSingh07** [comment]
Does this issue persist following the latest commit @NiranjanNlc @AvneetSingh2001 ? If not then we can close this issue and the PR linked to this

**Comment by AvneetSingh2001** [comment]
I have migrated the HomeFragment to compose and handled this issue already in that PR, which is merged also.
But for navigation drawer, the issue is still there.  @PratyushSingh07
One way is @NiranjanNlc  can only include the changes of Navigation drawer in the PR, or we can raise another PR to migrate drawer also to Compose.

**Comment by PratyushSingh07** [comment]
Thank you @AvneetSingh2001 . @NiranjanNlc the choice is yours. You can either migrate the nav drawer to compose or you can amend your existing PR to only include the changes in nav drawer.
If you ask me then migrating the nav drawer to compose will be a more viable option

**Comment by NiranjanNlc** [comment]
@PratyushSingh07 , ammended the PR for now .
Needed some more effort, time and learning to migrate nav drawer .
please get my PR reviewed for now .

**Comment by therajanmaurya** [comment]
No longer a issue
